<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>442</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>690</width>
    <height>442</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>690</width>
    <height>442</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Monorotor Commander 3</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_model">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>341</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="pushButton_add">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>В очередь</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_singleStart">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>270</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_model">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>241</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <widget class="QWidget" name="tab_params">
      <attribute name="title">
       <string>Основные</string>
      </attribute>
      <widget class="QFrame" name="frame_params">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>311</width>
         <height>201</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_ratioAB">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Соотношение A:B</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_volume">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Объём, мкл</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ratioA">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_reverse">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>90</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Реверс, мкл</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_feedrate">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>50</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Подача, мкл/мин</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_dozators">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_feedrate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ratioB">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>51</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_reverse">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_volume">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>7</number>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_details">
      <attribute name="title">
       <string>Расширенные</string>
      </attribute>
      <widget class="QFrame" name="frame_details">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>311</width>
         <height>161</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit_accel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_accel">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>10</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ускорение, мкл/минсек</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_gearA">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>40</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Коэффициент А</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_gearA">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_gearB">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhDigitsOnly</set>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_gearB">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Коэффициент B</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_dir">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>211</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Обратное направление</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_specChange">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_change">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_order">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>321</width>
      <height>301</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QPushButton" name="pushButton_start">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>270</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Пуск</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_delete">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_reset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Сброс</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableView_order">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>300</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="gridStyle">
      <enum>Qt::SolidLine</enum>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>60</number>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>21</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_com">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QComboBox" name="comboBox_com">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_com">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Подключить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_com">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Выберите COM-порт</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
